### ISSUE TITLE
Is the new "userecho" page linked on the astropy homepage/documentation?

### ISSUE THREAD

**Issue by MSeifert04** [astropy/astropy#4673]
Only per accident I stumbled onto http://astropy.userecho.com/ ... I think it is still new and experimental but shouldn't it be linked in some obvious place on the homepage or in the documentation? But I couldn't find it there. 

For example http://www.astropy.org/contribute.html would be a very obvious place to put it.

**Comment by pllim** [comment]
Is this a duplicate of #1457?

**Comment by MSeifert04** [comment]
Actually no but it's related. #1457 is about starting such a system - since it is already started see link above that one could be closed?

This issue is more about having someone who visits the astropy homepage and wants to propose a feature to find a link to the *existing* userecho page: http://astropy.userecho.com/

**Comment by pllim** [comment]
I think @astrofrog is in the best position to comment on this.

**Comment by astrofrog** [comment]
The site is not properly public yet, but once it is, we should indeed link to it from the homepage.

**Comment by astrofrog** [comment]
@MSeifert04 - just out of curiosity, you mentioned that you found this by accident - are you not subscribed to astropy-dev? (I sent an email about it a few days ago)

**Comment by MSeifert04** [comment]
@astrofrog - Yes I remembered reading something somewhere about it. But I couldn't remember where and currently I've disabled notifications on dev because of the GSoC mails there. But then I tried to find it again and only after several failed attempts I located it. 

That was what inspired me to ask if it could be linked somewhere on the homepage.

**Comment by astrofrog** [comment]
@MSeifert04 - ok, thanks for clarifying!

**Comment by pllim** [comment]
@astrofrog , since this was officially mentioned in PyAstro16, should we go ahead and do this?

**Comment by astrofrog** [comment]
Yes, we should add this to the website somewhere. I would actually suggest adding it just below 'Please remember to acknowledge the use of Astropy!' in a sentence that says 'You can now suggest and vote on features for astropy!'. What do you think @eteq?

**Comment by MSeifert04** [comment]
Is the link now included? And if so, where?

cc @astrofrog @eteq

**Comment by astrofrog** [comment]
I've opened https://github.com/astropy/astropy.github.com/issues/142 to make sure we include a prominent link on the website, since that's the right issue tracker for this to be in.
